docs/assets/css/talis.css

```diff
@@ -1,7 +1,3 @@
der-color:#017d87;-webkit-box-shadow:0 0 0 1px #017d87;box-shadow:0 0 0 1px #017d87}.pagination{font-size:1.015625rem;font-variant-numeric:proportional-nums}.pagination .page-link:focus{border-color:#017d87;-webkit-box-shadow:0 0 0 1px inset #017d87;box-shadow:0 0 0 1px inset #017d87}.pagination>.active>.page-link{font-weight:600;color:#017d87;background-color:#fff;background-image:linear-gradient(0deg, #017d87 4px, transparent 4px);border-color:#017d87}.progress{border:1px solid #017d87}.o-skip-link:not(:focus):not(:focus-within){position:absolute !important;width:1px !important;height:1px !important;padding:0 !important;margin:-1px !important;overflow:hidden !important;clip:rect(0, 0, 0, 0) !important;white-space:nowrap !important;border:0 !important}.c-skip-link:focus,.o-skip-link:focus{position:absolute;top:0;left:0;padding:1rem 2rem;font-weight:600;color:#fff;text-decoration:underline;background-color:#005d64;outline:3px solid #fff;opacity:initial;-webkit-transition:none;transition:none;text-underline-position:under}.table>:not(caption):not(colgroup)+*{border-top:2px solid #e9ecef !important}th{color:#000}.table-lg>:not(caption)>*>*{padding:1.25rem 1.25rem}.toast-container{z-index:1070}.toast{margin-bottom:.75rem;font-weight:300}.toast-success{background-color:#d0e6dd}.toast-info{background-color:#a4d1ea}.toast-warning{background-color:#fed989}.toast-danger{background-color:#f3d5d5}.toast .btn-close{width:auto;height:auto;padding:0;font-weight:300;text-decoration:underline;background:transparent;opacity:1;text-underline-offset:1px}.toast .btn-close:hover,.toast .btn-close:focus{text-decoration:none}.toast>*:only-child{-webkit-box-align:center;-ms-flex-align:center;align-items:center;padding:.75em 1.5em}.toast-body:not(:only-child){padding:0 .75em}h1,h2,h3,h4,h5,h6,.h1,.h2,.h3,.h4,.h5,.h6,.alpha,.beta,.gamma,.delta,.epsilon,.zeta{line-height:1.1}h1,.h1,h2,.h2,h3,.h3{margin-top:20px;margin-bottom:10px}h4,.h4,.delta,h5,.h5,.epsilon,h6,.h6,.zeta{margin-top:10px;margin-bottom:10px}h1,.h1,.alpha,h3,.h3,.gamma,h5,.h5,.epsilon{color:#000}h2,.h2,.beta,h4,.h4,.delta{color:#868e96}h5,.h5,.epsilon,h6,.h6,.zeta{font-weight:700}h6,.h6,.zeta{color:#495057}.page-header{padding-bottom:9px;margin:40px 0 20px;border-bottom:1px solid #e9ecef}a:focus{outline:1px solid currentColor}body{-webkit-font-smoothing:antialiased}.o-list--striped>li:nth-child(odd){background-color:#f8f9fa}.well{min-height:20px;padding:12px;margin-bottom:20px;background-color:#f8f9fa;border:1px solid #e9ecef}.o-confirmation-modal .modal-dialog{max-width:26.25rem}.o-confirmation-modal .modal-image{margin:.9375rem .9375rem 0}.o-confirmation-modal .modal-header{-webkit-box-pack:start;-ms-flex-pack:start;justify-content:flex-start;padding-bottom:0}.o-confirmation-modal .modal-content{padding:.46875rem;border-radius:.5rem;font-size:1rem}.o-confirmation-modal .modal-footer{-webkit-box-pack:start;-ms-flex-pack:start;justify-content:flex-start;background-color:transparent;border-color:transparent}
```

Comment:
This file has been regenerated - it was frozen in time because the sources that were historically used to generate it were removed in [this commit](https://github.com/techfromsage/bootstrap-theme/commit/f11e808cc50346cf2aea1ed203519ed6e2ea07a3#diff-37436a76c8ce0232d27b6fdc8083147694966e8c543eb91eba3a7eb7dc3642c2).

I don't think Dan realised he had orphaned this file, nor how it would be used in future. Hence I've reestablished the link and documented it.